src/ui/public/saved_objects/find_object_by_title.ts

```diff
@@ -18,6 +18,9 @@
  */
 
 import { find } from 'lodash';
+import { SavedObjectAttributes } from 'src/server/saved_objects';
```

Comment:
I discussed that with the platform team (and besides that import actually being wrong, because it needs to be relative), it sounds to be the way to go atm.